### DISCUSSION TITLE
0.7.1? release timing

### DISCUSSION THREAD

**Discussion by laingsimon** [ItsBenCodes/cookies#691] [General]
Hi

Can you please release, or indicate when you would release the next version of the universal-cookie module. There is a [CVE related to the use of cookie <0.7.0](https://github.com/advisories/GHSA-pxg6-pf52-xh8x) which [has been fixed](https://github.com/bendotcodes/cookies/commit/0d565c661b3ce0bee8129491f349b36599c1c502) but which hasn't yet been released.

`npm audit` is reporting the vulnerability which impacts the builds of a project i'm working on:

```
# npm audit report

cookie  <0.7.0
cookie accepts cookie name, path, and domain with out of bounds characters - https://github.com/advisories/GHSA-pxg6-pf52-xh8x
fix available via `npm audit fix --force`
Will install react-cookie@1.0.5, which is a breaking change
node_modules/universal-cookie/node_modules/cookie
  universal-cookie  *
  Depends on vulnerable versions of cookie
  node_modules/universal-cookie
    react-cookie  >=2.0.1
    Depends on vulnerable versions of universal-cookie
    node_modules/react-cookie
```

**Comment by laingsimon** [comment]
Issue has been raised #693

**Comment by ItsBenCodes** [comment]
The v7.2.1 has been released and should fix the security issue.
